main.js

```diff
@@ -49,21 +74,21 @@ function createIdeaCard() {
   
   newIdea = new Idea(title, body)
   allIdeas.push(newIdea)
-  displayAllIdeas()
+  displayIdeas(allIdeas)
 }
 
-function deleteIdeaCard(event) {
+function deleteIdeaCard(array) {
```

Comment:
I think you could rename the argument into the deleteIdeaCard function to idea for clarity. And maybe you could call it jus the deleteCard function and pass in the (ideas) or (starredIdeas) you want to delete. What do you think?